src/test/java/org/isf/patient/test/Tests.java

```diff
@@ -96,14 +97,14 @@ public void testIoGetPatients() throws Exception {
 		List<Patient> patients = patientIoOperation.getPatients();
 		testPatient.check(patients.get(patients.size() - 1));
 	}
-
-	@Test
+	
+	@Test 
 	public void testIoGetPatientsPageable() throws Exception {
-		setupTestPatient(false);
-		List<Patient> patients = patientIoOperation.getPatients(createPageRequest());
-		testPatient.check(patients.get(patients.size() - 1));
+	  setupTestPatient(false); 
```

Comment:
the previous formatting looks more correct.   Why is there a trailing blank added on line 103 and 105?